pyk/src/pyk/klean/template/{{ cookiecutter.package_name }}/{{ cookiecutter.library_name }}/Prelude.lean

```diff
@@ -210,3 +210,8 @@ class Inj (From To : Type) : Type where
   inj (x : From) : To
 
 def inj {From To : Type} [inst : Inj From To] := inst.inj
+
+def «_+Int_» (x0 : SortInt) (x1 : SortInt) : Option SortInt := some (x0 + x1)
+def «_-Int_» (x0 : SortInt) (x1 : SortInt) : Option SortInt := some (x0 - x1)
+def «_*Int_» (x0 : SortInt) (x1 : SortInt) : Option SortInt := some (x0 * x1)
+def «_<=Int_» (x0 : SortInt) (x1 : SortInt) : Option SortBool := some (x0 <= x1)
```

Comment:
Shouldn't other functions be added? Such as `«_modInt_»`? We could also add hooked functions such as `_List_`. They would be `noncomputable` rather than the current `axiom`s, and we have the above hooks to link to them.

Reply:
Yes, these few functions are just to demonstrate how to define functions that have no associated function rules:
* Add function definition to `Preulde`
* Add KORE symbol name to `_PRELUDE_FUNCS` in `k2lean4.py`.